fenix/app/src/main/res/values/strings.xml

```diff
@@ -2302,6 +2302,65 @@
 
     <!-- Translations feature-->
 
+    <!-- Translations bottom sheet -->
+    <!-- Title of the translations bottom sheet-->
+    <string name="translations_bottom_sheet_title">Translate this page?</string>
+    <!-- Title of the translations bottom sheet. It will appear only once when the user uses the first-time translation feature.
+    The first parameter is the name of the app defined in app_name (for example: Fenix) -->
+    <string name="translations_bottom_sheet_title_first_time">Try private translations in %1$s</string>
+    <!-- Translations bottom sheet info message. It will appear under the title.It will appear only once when the user uses the first-time translation feature.
```

Comment:
```suggestion
    <!-- Translations bottom sheet info message. It will appear under the title. It will appear only once when the user uses the first-time translation feature.
```